docs/technical_reference/flowsheets/extended_BSM2.rst

```diff
@@ -204,6 +207,11 @@ Additional Constraints
    "Reactor 5 mass transfer", ":math:`injection_{R5, S_{O2}} = KLa_{R5} * V_{R5} * (S_{O, eq} - S_{O, out})`"
    "Reactor 6 mass transfer", ":math:`injection_{R6, S_{O2}} = KLa_{R6} * V_{R6} * (S_{O, eq} - S_{O, out})`"
    "Reactor 7 mass transfer", ":math:`injection_{R7, S_{O2}} = KLa_{R7} * V_{R7} * (S_{O, eq} - S_{O, out})`"
+   "Total suspended solids concentration", ":math:`TSS_{out} <= TSS_{max}`"
+   "Chemical oxygen demand", ":math:`COD_{out} <= COD_{max}`"
+   "Total phosphorus concentration", ":math:`P_{out} <= P_{max}`"
+   "Total nitrogen concentration", ":math:`N_{out} <= N_{max}`"
+   "5-day biological oxygen demand", ":math:`BOD5_{out} <= BOD5_{max}`"
```

Comment:
Should this be removed until we are able to incorporate as constraints?

Reply:
Sure - I'll get rid of the change in this PR and leave it in the follow-up PR (1503)